Zend/tests/debug_info-error-anon-class.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+Testing __debugInfo() magic method with invalid returns inside anon-class
```

Comment:
Note for reviewers: I've added this test to ensure that the error message is `Method C::__debugInfo() ...` instead of `Method class@anonymous::__debugInfo() ...`.